content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst

```diff
@@ -0,0 +1,37 @@
+=================
+Payment providers
+=================
+
+Odoo supports a multitude of online
+:doc:`payment providers <../../../../applications/finance/payment_providers>` for your website,
+allowing your customers to pay with their preferred payment methods.
+
+.. seealso::
+   :doc:`/applications/finance/payment_providers`
+
+Configuration
+-------------
+
+To set up payment providers on the eCommerce app, go to :menuselection:`Website --> Configuration
+--> Payment Providers`. From here, :guilabel:`Activate` the payment providers you wish to have
+available on your shop, and configure them according to your needs.
+
+Alternatively, you can access **payment providers** via :menuselection:`Website --> Configuration
+--> Settings`. In the :guilabel:`Shop - Payment` section, you can :guilabel:`Configure SEPA Direct
+Debit` if you wish to use it, as well as :guilabel:`View other providers`. If you use the
+:guilabel:`Authorize.net` payment provider, the
+:ref:`Payment Capture Method <payment_providers/features/manual_capture>` can be configured in that
+same menu.
+
+If you are using :doc:`../../../finance/payment_providers/paypal`, you can also enable and configure
```

Comment:
Here too, but in this case it's correct like this if you prefer to use relative refs over absolute refs.